feature/dashboard/src/main/java/com/simprints/feature/dashboard/debug/DebugFragment.kt

```diff
@@ -47,7 +50,7 @@ internal class DebugFragment : Fragment(R.layout.fragment_debug) {
     lateinit var eventRepository: EventRepository
 
     @Inject
-    lateinit var enrolmentRecordRepository: EnrolmentRecordRepository
+    lateinit var enrolmentRecordRepository: EnrolmentRecordLocalDataSource
```

Comment:
Not that it matters but why? :)

Reply:
The local data source does not need the "BiometricDataSource" arg :D It is resolved to the local source within the repo anyway.